src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if (APPLE)
```

Comment:
If there's any intention of this making it into v1/RTM, we need to do things right the first time.  There's no "later" to defer problems to.